output/influxdb/config.go

```diff
@@ -17,6 +17,7 @@ import (
 type Config struct {
 	// Connection.
 	Addr             null.String        `json:"addr" envconfig:"K6_INFLUXDB_ADDR"`
+	Proxy            null.String        `json:"proxy,omitempty" envconfig:"K6_INFLUXDB_PROXY"`
```

Comment:
```suggestion
	Proxy            null.String        `json:"proxy,omitempty" envconfig:"K6_INFLUXDB_HTTP_PROXY"`
```
@mstoykov WDYT? Should we do it? I like the consistency with the original HTTP_PROXY variable, but on the other side, I'm not sure because it could introduce confusion around the expected HTTP/S value.

Reply:
it's because when I set HTTP_PROXY and HTTPS_PROXY vars, k6 sends all request through proxy, not just output results.

Reply:
Added a config test for the K6_INFLUXDB_PROXY variable. I will add a test for the proxy connectivity as well.